src/util/paths.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+const repoRoot = process.cwd(); // expect CLI to be run from consumer repo root
+const appPath = path.resolve(repoRoot, 'src');
+
+module.exports = {
+	repoRoot,
+	appPath,
+};
```

Comment:
I think it would be good to consolidate this with `src/commands/buildCommands/config/paths` so we don't have two sources for that info, but I'd like your take - should we make an effort to split build-relevant `paths` settings  (e.g. entry/output for bundles) from more 'global' paths?